enforcer-parent/commons/src/main/java/org/wso2/choreo/connect/enforcer/commons/logging/CustomJsonLayout.java

```diff
@@ -63,6 +63,7 @@ public String toSerializable(LogEvent event) {
         obj.put(LoggingConstants.LogAttributes.LEVEL, event.getLevel().toString());
         obj.put(LoggingConstants.LogAttributes.LOGGER, event.getLoggerName());
         obj.put(LoggingConstants.LogAttributes.MESSAGE, event.getMessage().getFormattedMessage());
+        obj.put(LoggingConstants.LogAttributes.CONTEXT, event.getContextStack().asList());
```

Comment:
For the JSON format, we can include the traceId as a separate field.

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2679/commits/356dd76f69e350e032432200814911fd84269ee1